WSLGd/main.cpp

```diff
@@ -393,6 +401,15 @@ try {
     westonShellOption += westonShellName;
     westonShellOption += ".so";
 
+    // Construct log file option string.
+    std::filesystem::path westonLogFileOption("--log=");
```

Comment:
nit: it feels a bit unnatural to have a std::filesystem::path start with `--`, maybe westonLogFileOption should be a std::string instead ?

Reply:
right, while it's file path, but it is argument. Changed.